modules/fedimint-lnv2-common/src/gateway_api.rs

```diff
@@ -188,11 +192,9 @@ impl RoutingInfo {
             (self.send_fee_minimum.clone(), self.expiration_delta_minimum)
         } else {
             (
-                PaymentFee {
-                    base: self.send_fee_default.base + self.send_fee_minimum.base,
-                    parts_per_million: self.send_fee_default.parts_per_million
-                        + self.send_fee_minimum.parts_per_million,
-                },
+                self.send_fee_default
+                    .clone()
+                    .add(self.send_fee_minimum.clone()),
```

Comment:
You can not change client side behavior like this. This addition needs to be done on the gateways side instead.

Reply:
Fixed, moved to addition to the gateway